app/controllers/api/v1/opportunities_controller.rb

```diff
@@ -13,8 +27,77 @@ def show
       render json: @opportunity, status: :ok
     rescue ActiveRecord::RecordNotFound => e
       render json: {
-          error: e.to_s
-      }, status: :not_found
+        opportunity_id: params[:id],
+        error: e.to_s,
+        status: 'Opportunity not found'
```

Comment:
@mwharrison My thought was that since the app does not have a frontend, returning a 404 is not helpful. Returning json with a status and error message would be best. I can put the 404 back.

Reply:
Ah, I think we will need a 404. The Swagger tests depend on them - and there will be a FE eventually

Reply:
@mwharrison Restored the 404